src/coreclr/jit/gentree.cpp

```diff
@@ -11002,9 +11133,60 @@ void Compiler::gtDispConst(GenTree* tree)
                 printf(" %#.17g", tree->AsDblCon()->gtDconVal);
             }
             break;
+
         case GT_CNS_STR:
             printf("<string constant>");
             break;
+
+        case GT_CNS_VEC:
+        {
+            GenTreeVecCon* vecCon = tree->AsVecCon();
+
+            switch (vecCon->TypeGet())
+            {
+#if defined(FEATURE_SIMD)
+                case TYP_LONG:
+                case TYP_DOUBLE:
+                case TYP_SIMD8:
+                {
+                    // TYP_SIMD8 may get retyped to TYP_LONG or TYP_DOUBLE in lowering or morph
```

Comment:
`TODO-Cleanup: do not retype SIMD nodes, turn them into appropriate nodes of the other type instead.`